cmd/binlog-collector/storage/storage.go

```diff
@@ -0,0 +1,69 @@
+package storage
+
+import (
+	"context"
+	"io"
+	"io/ioutil"
+
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+	"github.com/pkg/errors"
+)
+
+// Manager is a type for working with S3 storages
+type Manager struct {
```

Comment:
> Manager is a type for working with S3 storages
So why it called Manager but not S3 let's say? Manager to general name - it says nothing to the user.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/622/commits/cec0c704c6c8cb2eb0957e2123bd8b04478b8605